Tasks/ServiceFabricDeploy/Create-DiffPackage.psm1

```diff
@@ -142,6 +142,14 @@ function Copy-DiffPackage
 
         $localPkgPath = Join-Path $localParentPkgPath $localPackage.Name
         $diffPkgPath = Join-Path $diffParentPkgPath $localPackage.Name
+        $localZipPkgPath = $localPkgPath + ".zip"
```

Comment:
the new variables are not really necessary. you can directly use `$localPkgPath + ".zip"` later.